README.md

```diff
@@ -28,7 +30,7 @@ Please read the license terms applicable to this app template [here](https://git
 
 * Microsoft will have no access to data collected through your app.  Microsoft will not be responsible for any data related incidents or data subject requests.
 
-* Any trademarks or registered trademarks of Microsoft in the United States and/or other countries and logos included in this repository are the property of Microsoft, and the license for this project does not grant you rights to use any Microsoft names, logos or trademarks outside of this repository.  Microsoft�s general trademark guidelines can be found [here](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general.aspx)
+* Any trademarks or registered trademarks of Microsoft in the United States and/or other countries and logos included in this repository are the property of Microsoft, and the license for this project does not grant you rights to use any Microsoft names, logos or trademarks outside of this repository.  Microsoft’s general trademark guidelines can be found [here](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general.aspx)

```

Comment:
What did you change here?

Reply:
weird. it didn't showed up in GitHub and IDE as difference. I believe the encoding got changed to UTF-8 

From git diff
**Microsoft's** text before Microsoft<92>s changed to Microsoft<E2><80><99>s. No changes in actual text.